oi.py

```diff
@@ -1,39 +1,86 @@
 import wpilib
 from wpilib import buttons
 
+from robot_map import RobotMap
+"""
+ROBOT AXIS we use these axis mappings, which are different from the wpilib ones
+      + X
+        ^
+        |
++ Y <------->
+        |
+        V
+Z axis pointing up, with a +ve rotation beting counter clockwise (right hand rule)
+
+WPILIB AXIS MAPPINGS - used when getting axis from a joystick or gamepad
+
+        ^
+        |
+    <------> + X
+        |
+        V
+        +Y
+"""
```

Comment:
Same as before. It is better to capture the approach in an external document. It is not ideal to explain a lot of things in the code itself. If we change our approach, we will have to spend time to fixing them